packages/core/src/test/codewhisperer/service/transformByQ/humanInTheLoopHandler.test.ts

```diff
@@ -0,0 +1,150 @@
+/*!
```

Comment:
TODO: Rename this file since we have a humanInTheLoop manager class. These should be refactored into a utilities file test or something.